lib/services/access_list.go

```diff
@@ -166,7 +166,13 @@ func UnmarshalAccessListMember(data []byte, opts ...MarshalOption) (*accesslist.
 }
 
 // IsOwner will return true if the user is an owner for the current list.
+// TODO(mdwn): Remove this.
```

Comment:
Is this being kept for now so `e` builds don't fail?

Reply:
Good point. Can you add more context? Why this cannot be removed now? Should be removed on the next release...tomorrow?

Reply:
Yes it is. I'll add more context here.